examples/js/exampleApp.js

```diff
@@ -8,6 +8,13 @@
 
     var data = this.data = {};
 
+    data.nameTokens = {
+      '[CURRENT_USER]': 'Current User',
+    };
+    data.mailTokens = {
+      '[CURRENT_USER]': 'active@user.com',
+    };
+
```

Comment:
because it is using for many places so it is better if we can create a new variable. I added to `data` so that it can be consistent with other settings

Reply:
This project does not need to know every places that use it. Try to make thing simplier e.g.
```
'test.term': { field: 'first_name', title: 'First Name', type: 'term', placeholder: 'My First Name', helptext: 'Do not put your last name here' },
```
  So those variables are not needed.